docs/document/static/js/samples/desktop/user-org-group-select.jsx

```diff
@@ -0,0 +1,55 @@
+import React, { useEffect, useRef } from "react";
+
+export const UserOrgGroupSelectComponent = () => {
+  const divEl = useRef();
+  useEffect(() => {
+    const userOrgGroupSelect = new Kuc.UserOrgGroupSelect({
+      label: "Sample label",
+      requiredIcon: true,
+      items: [
+        {
+          label: "User A",
+          value: "userA",
+          type: "user",
+          disabled: true,
+        },
+        {
+          label: "User B",
+          value: "userB",
+          type: "user",
+        },
+        {
+          label: "Organization AOrganization AOrsAOrganization",
+          value: "orgA",
+          type: "org",
+        },
+        {
+          label: "Group A",
+          value: "groupA",
+          type: "group",
+        },
+        {
+          label: "User C",
+          value: "userC",
+          type: "user",
+        },
+      ],
+      value: ["userB", "orgA", "groupA"],
+      error: "Error occurred!",
+      className: "sample-class",
+      icon: "user",
+      id: "sample-id",
+      placeholder: "Sample placeholder",
```

Comment:
Please use actual use case.

```suggestion
      placeholder: "Please select assignees",
```